packages/webview_flutter/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: webview_flutter
 description: A Flutter plugin that provides a WebView widget on Android and iOS.
-version: 0.3.5+2
+version: 0.3.5+3
```

Comment:
As this is adding a new feature Y in 0.X.Y+Z should be bumped, please also update the version in the changelog

Reply:
fixed